ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -111,21 +156,29 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'P4S Kothornos Kick',
       regex: /Kothornos Kick/,
-      beforeSeconds: 5,
+      beforeSeconds: 5.3,
```

Comment:
This is a 5 second cast.  Should this be a normal trigger and not a timeline trigger?

Reply:
The second one requires timeline trigger or duplicate trigger with delay as the second one is instant cast